src/fromager/sources.py

```diff
@@ -119,6 +140,136 @@ def resolve_source(
     return str(url), version
 
 
+def resolve_version_from_git_url(
+    ctx: context.WorkContext,
+    req: Requirement,
+) -> tuple[str, Version]:
+    "Return path to the cloned git repository and the package version."
+
+    if not req.url.startswith("git+"):
+        raise ValueError(f"do not know how to handle URL scheme in {req}")
```

Comment:
Maybe change it to "Can't handle URL scheme in {req} because of {req.url}" ? Or is it too verbose than we want to be?

Reply:
I had a version of the message with the URL pulled out originally and removed it, but I can put that back. I am 99% sure that the requirement is rendered with the URL included, but if I pull it out I can quote it so it stands out.

Reply:
Okay, sounds good!

Reply:
Fixed